### ISSUE TITLE
Black-on-black title bar text is difficult to read (color contrast)

### ISSUE THREAD

**Issue by trueriver** [QubesOS/qubes-issues#3854]
### Qubes OS version:

R4.0 (and probably earlier)

### Affected component(s): 

Whichever component sets the colour of title bars of VM windows

---

### Steps to reproduce the behavior:

Open a Template VM (assuming default colour settings)

### Expected behavior:

Legible title in the Title bar

### Actual behavior:

Low contrast title in the Tile bar: black on slightly-lighter black.

### General notes:

My suggestion is to either to hard code a special case for black, or to code a test that if the monochrome brightness of the title bar colour is lower that 50%, change the text colour to white or yellow.

My preference would be yellow, because ergonomic tests show (surprisingly) that yellow shows up better than white against a black backround

---

**Comment by andrewdavidwong** [comment]
Are you using Xfce4, the default desktop environment?

**Comment by ThomasWaldmann** [comment]
I can confirm the issue with qubes 4.0.1-rc1, using the default xfce4 DE.

The problem also applies to the "vault" AppVM as it uses the same colour combination.

I just wanted to file a bug about it, but found this one.

**Comment by gatl** [comment]
I can confirm the issue persists with Qubes OS 4.0.1-rc2, on the default Xfce4 environment.
And it is annoying.
